### ISSUE TITLE
Edit with form not showing on category pages

### ISSUE THREAD

**Issue by krabina** [gesinn-it-pub/mediawiki-extensions-PageForms#73]
I am using a Form:Foo to create a page in the category namespace. The form creates a page called Kategorie:Bar.
On this page, template Baz is added by PageForms. In the template Baz the call `{{#default_form:Foo}}` is added.

What works on regular pages (in the main namespace) does not work now: there is no "edit wit form" tab. There is no way to go back to the form, other than typing the page name into the form.

The only solution is to put  `{{#default_form:Foo}}` in the Projekt:Kategorie page. But this leads to all category pages being pointed to a form, which is not good, since it does not make sense in all cases.

IMO `{{#default_form:Foo}}` should work also on category pages. In fact, it should even override anything set in the project namespace setting.

**Comment by gesinn-it-gea** [comment]
@krabina did understand this correctly: you create Categories (e.g. "Bar") using "From:Foo" that adds template "Baz" to the Category page. Inside "Baz", you set {{#default_form:Foo}}.

Did you try having all those categories created by Form:Foo being subcategory of one Category, e.g. "Category:Foo" and adding the {{#default_form:Foo}} there?

**Comment by krabina** [comment]
Yes, that is correct. Adding {{#default_form:Foo}}  to a category page just tells PageForms that all pages belonging to this category should be opened with the form.
However, I need the category page itself to be opened with the form. Which should not be a problem, since {{#default_form:Foo}}  ist in the Template "Baz". 
But for whatever reason, this does not work.
